gci/config.py

```diff
@@ -27,6 +29,7 @@ def get_api_key(name):
             api_key = api_key_file.readline().strip()
             return api_key
     except IOError:
+        logger.error('Please put your %s API key at %s.' % (name, filename))
```

Comment:
as a general rule, do not log before an exception.

An exception is telling the caller there was a problem, and the caller then has to decide what to do with the exception.
The caller may already know about this type of exception, and decide to ignore it because they do not consider it to be an error.

Reply:
ok